common/flatpak-context.c

```diff
@@ -841,6 +843,9 @@ parse_filesystem_flags (const char            *filesystem,
   if (mode_out)
     *mode_out = mode;
 
+  if (explicit_suffix_out)
+      *explicit_suffix_out = (suffix != NULL);
```

Comment:
Please indent this consistently: Flatpak uses GNU-style indentation. The block that handles `mode_out` is correct.